app/pages/project/networking/VpcPage/tabs/VpcFirewallRulesTab.tsx

```diff
@@ -32,10 +33,17 @@ import { confirmDelete } from 'app/stores/confirm-delete'
 
 const colHelper = createColumnHelper<VpcFirewallRule>()
 
+const directionMap = {
+  inbound: 'Incoming',
+  outbound: 'Outgoing',
+}
```

Comment:
We should use the API's language where possible. In this case the disparity feels unnecessary.

Reply:
Ah I agree, thought this came from
you and was matching the form. In that case I should probably tweak the form too?

Reply:
Yep, fix it